airflow/config_templates/default_airflow.cfg

```diff
@@ -537,6 +537,9 @@ ssl_cacert =
 # https://docs.celeryproject.org/en/latest/userguide/concurrency/eventlet.html
 pool = prefork
 
+# The number of seconds to wait before timing out apply_async
+connection_timeout = 2
```

Comment:
Is it actually a `connection_timeout`? Trying to think if we can call it something specific to apply_async

Reply:
@ashb what do you think?

Reply:
You are right. To be precise, this will become the timeout for `fetch_celery_task_state()` and `send_task_to_executor()`. I could not come up with a perfect name for it. I'm open to suggestions.

Reply:
@kaxil @ashb  I renamed it to `operation_timeout` because it's the timeout for the `send_task_to_executor()` and `fetch_celery_task_state()` operations. Please let me know if you have better suggestions.